paddle/fluid/operators/math/softmax_impl.h

```diff
@@ -27,7 +27,7 @@ using EigenMatrix = framework::EigenMatrix<T, MajorType, IndexType>;
 template <typename T>
 struct ValueClip {
   HOSTDEVICE T operator()(const T& x) const {
-    const T kThreshold = -64.;
+    const T kThreshold = static_cast<T>(-64.);
```

Comment:
Do we need a comment here about the criteria for choosing the value of the threshold?